exec/java-exec/src/main/resources/rest/index.ftl

```diff
@@ -53,11 +53,16 @@
             <tr>
               <th>#</th>
               <th>Address</th>
+              <th>Heap Usage</th>
+              <th>Direct Usage</th>
               <th>User Port</th>
               <th>Control Port</th>
               <th>Data Port</th>
               <th>Version</th>
               <th>Status</th>
+              <#if model.shouldShowAdminInfo() || !model.isAuthEnabled() >
```

Comment:
On master we have different line `<#if (model.shouldShowAdminInfo() || !model.isAuthEnabled()) && drillbit.isCurrent() >`

Reply:
I need to rebase. DRILL-6044 fix changed that entry.